iocore/aio/AIO.cc

```diff
@@ -587,7 +576,13 @@ DiskHandler::mainAIOEvent(int event, Event *e)
   }
 
   while ((op = complete_list.dequeue()) != nullptr) {
-    op->handleEvent(event, e);
+    op->mutex = op->action.mutex;
```

Comment:
I think aio should consider the user's lock.

Reply:
I'm moving it into `io_complete`.

Reply:
Yes you are right ! the original code locked the CacheVC.